pypeit/specobj.py

```diff
@@ -25,6 +25,7 @@
 from pypeit import utils
 from pypeit import datamodel
 from pypeit.images import detector_container
+from pypeit.spectrographs.util import load_spectrograph
```

Comment:
I think this can be removed since you don't require spectrograph.

Reply:
agreed; gone